README

```diff
@@ -8,12 +8,11 @@ Tengine has been an open source project since December 2011. It is being activel
 
 Features
 --------
-* All features of nginx-1.9.15 are inherited, i.e., it is 100% compatible with nginx.
+* All features of nginx-1.15.9 are inherited, i.e., it is 100% compatible with nginx.
 * Support the CONNECT HTTP method. could be used in forward proxy scenarios.
```

Comment:
-> `Support the CONNECT HTTP method for forward proxy`

Reply:
fixed.